Indentation

From what we can see from the configuration of [other languages](https://github.com/search?q=path%3Alanguages%2F*%2Findents.scm&type=code), indentation rules should be provided in a `languages/rego/indents.scm` file, using tree-sitter queries. The contents of this directory, and the `indents.scm` file is not described in the Zed docs. In fact the _only_ documentation provided on the `languages` directory is this single sentence:

<img width="549" alt="Screenshot 2024-05-29 at 00 36 38" src="https://github.com/StyraInc/zed-rego/assets/510711/67b4e238-7511-41d8-a2ab-a0bd93f7230b">

Ugh.

But anyway, again looking at how other languages are configured, it would seem like a good starting point, and likely even good enough, to just increase indentation on `{`, `[` and `(`, and decrease it on their closing counterparts. For a language like Clojure, this is [represented](https://github.com/zed-industries/zed/blob/main/extensions/clojure/languages/clojure/indents.scm) as:

```clojure
(_ "[" "]") @indent
(_ "{" "}") @indent
(_ "(" ")") @indent
```

Putting that in an a `languages/rego/indents.scm` file however breaks the extension on installation. From the logs:

```
2024-05-29T08:46:13+02:00 [ERROR] failed to load language rego:
Error loading indents query

Caused by:
    Query error at 1:5. Invalid node type 
```
 
I've tried all of the alternatives I've found in the different language configurations, but the result is either an error, or indentation increase working but not decrease.

--- a/README.md
+++ b/README.md
@@ -4,13 +4,12 @@ Zed extension for Rego and [Open Policy Agent](https://github.com/open-policy-ag
 
 **This is work in progress and not yet ready for general use.**
 
-## Status
+## Features
 
-### Basic Rego Support
+### Rego Syntax Support
 
-Basic features like syntax highlighting works well, with the following exceptions:
-
-- [ ] No automatic indentation in rule bodies, objects, etc
+Basic features like syntax highlighting and indentation works out of the box. More advanced features are provided by
+[Regal](https://github.com/styrainc/regal).
 
 ### Regal Language Server Support
 
@@ -27,7 +26,7 @@ functionality. The following features of the language server are currently suppo
 - [x] Code actions
 - [x] Code completion
 
-## Installation
+## Development
 
 To install the extension in development mode, first make sure to have [Rust](https://www.rust-lang.org/tools/install) installed.
 

--- a/languages/rego/indents.scm
+++ b/languages/rego/indents.scm
@@ -1,0 +1,3 @@
+(_ (open_paren) (close_paren) @end) @indent
+(_ (open_curly) (close_curly) @end) @indent
+(_ (open_bracket) (close_bracket) @end) @indent

